src/Excepts.java

```diff
@@ -0,0 +1,64 @@
+import java.util.Random;
+
+// Custom checked exception
+public class NotEnoughGas extends Exception {
+    public NotEnoughGas(String message) {
+        super(message);
+    }
+}
+
+// Custom unchecked exception
+public class Explosion extends RuntimeException {
+    public Explosion(String message) {
+        super(message);
+    }
+}
+
+class Car {
+    private int fuel;
+    private int kilometers;
+    private Random random;
+
+    public Car() {
+        this.fuel = 35;
+        this.kilometers = 0;
+        this.random = new Random();
+    }
+
+    public void fill() {
+        if (random.nextInt(10) == 0) {
+            throw new Explosion("BOOOOOOM!");
+        }
+
+        // Return a random value from 15 to 36 (inclusive)
+        int randomFillOfFuel = random.nextInt(35 - 15 + 1) + 15;
```

Comment:
This code is correct, just the comment above is not.

'''
nextInt(int n) is used to get a random number between 0(inclusive) and the number passed in this argument(n), exclusive. 
'''

Comment says: 15 to 36 inclusive, means both 15 and 36 are included. But you don't want 36.
Change comment to: get random number from [15,35] (as in the exercise)

![image](https://github.com/user-attachments/assets/ac4b93ce-d89d-418d-9bcf-834029da3430)